backend/Tasque.BLL/Services/TaskService.cs

```diff
@@ -235,13 +247,21 @@ public async Task<List<TaskCustomFields>> MapCosmosTaskFieldsToTaskCustomFields(
             var template = await GetTaskTemplate(task.ProjectId, task.TypeId);
             var result = new List<TaskCustomFields>();
 
-            fields.ForEach(f => result.Add(new()
+            if (fields == null || fields.Count() == 0 || template == null || template.CustomFields == null)
```

Comment:
Use `template?.CustomFields == null`

Reply:
It's nnecessary coz previous condition checks if they're null

Reply:
`template?.CustomFields == null` will return `null` when `teamplate` is `null` due to the `?` operator